bootstrap/values.yaml

```diff
@@ -5,4 +5,29 @@ application:
   name: open-management-portal
   url: https://github.com/rht-labs/open-management-portal-deployment.git
   path: applications/
-  ref: master
\ No newline at end of file
+  ref: master
+
+argo-cd:
+  enabled: false
+  installCRDs: false 
```

Comment:
Does this mean that we need to install the `Application` CRD elsewhere? Or is this telling Argo whether or not it should install CRDs it encounters later?

Reply:
Nope, this is just a key that allows for helm v2 compatability. Long story short, they changed the way that CRD's were handled between v2 and v3 and this is how this particular set of charts is handling that.

Reply:
👍